content/spin/kv-store-tutorial.md

```diff
@@ -22,27 +20,14 @@ url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store.md"
 - [Conclusion](#conclusion)
 - [Next Steps](#next-steps)
 
-## Key Value Storage With Spin Applications
+## Key Value Store With Spin Applications
 
-Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times during an application’s execution. [Spin](https://www.fermyon.com/blog/spin-v09) has support for applications that need data in the form of key/value pairs and are satisfied by a BASE consistency model. Workload examples include general value caching, session caching, counters, and serialized application state.
+Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times during an application’s execution. [Spin](https://www.fermyon.com/blog/spin-v09) has support for applications that need data in the form of key/value pairs and are satisfied by a Basically Available, Soft State, and Eventually Consistent (BASE) model. Workload examples include general value caching, session caching, counters, and serialized application state. In this tutorial, you will learn how to do the following:
```

Comment:
Oh no, expanding the acronym means now I have even more questions (what is "soft state"?!?!?!).  We can defer that to a subsequent polishing pass though!